scripts/pylint_extensions.py

```diff
@@ -17,16 +17,32 @@
 """Implements additional custom Pylint checkers to be used as part of
 presubmit checks.
 """
+from __future__ import absolute_import
+from __future__ import unicode_literals
 
+import os
 import re
+import sys
+
 import astroid
-import docstrings_checker  # pylint: disable=relative-import
+import utils
 
 from pylint import checkers
 from pylint import interfaces
 from pylint.checkers import typecheck
 from pylint.checkers import utils as checker_utils
 
+from . import docstrings_checker
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_FUTURE_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'future-0.17.1')
+
+sys.path.insert(0, _FUTURE_PATH)
+
+# pylint: disable=wrong-import-position
+from builtins import range  # isort:skip
```

Comment:
Please add comment explaining why this needs to be here.

Reply:
done